event_webhook_types_test.go

```diff
@@ -230,23 +229,23 @@ func TestMergeEventUnmarshal(t *testing.T) {
 	}
 
 	if event.Assignees[0].AvatarURL != excpectedAvatar {
-		t.Errorf("Assignees[0].Email is %v, want %v", event.Assignees[0].AvatarURL, excpectedAvatar)
+		t.Errorf("Assignees[0].AvatarURL is %v, want %v", event.Assignees[0].AvatarURL, excpectedAvatar)
 	}
 
-	if event.Assignees[0].Email != expectedEmail {
-		t.Errorf("Assignees[0].Email is %v, want %v", event.Assignees[0].Email, expectedEmail)
```

Comment:
Please add this test back...

Reply:
I deleted the test, as the payload doesn't include an email address for the assignee anymore. Should I add fields to the payload, which deviate from the (current) documentation, instead?